<launch>

    <node pkg="b2b" type="pretend_bot.py" name="pretend_bot" />

    <node pkg="b2b" type="msg_parser" name="msg_parser" />
    <node pkg="b2b" type="ball_service.py" name="ball_service" />

</launch>
